pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstance.java

```diff
@@ -79,6 +84,18 @@ public JavaExecutionResult handleMessage(MessageId messageId, String topicName,
         return executionResult;
     }
 
+    public JavaExecutionResult handleMessage(Object input) {
```

Comment:
We should use the same handleMessage as above but pass null for messageId and topic